chia/cmds/start_funcs.py

```diff
@@ -56,11 +56,13 @@ async def async_start(root_path: Path, group: str, restart: bool) -> None:
                 continue
         print(f"{service}: ", end="", flush=True)
         msg = await daemon.start_service(service_name=service)
-        success = msg["data"]["success"]
+        success = msg and msg["data"]["success"]
 
         if success is True:
             print("started")
         else:
-            error = msg["data"]["error"]
+            error = "no response"
+            if msg:
+                error = msg["data"]["error"]
```

Comment:
Can even be cut down by two lines by putting the `msg` existence clause first, alla
```python
if msg:
    data = msg["data"]
    if data["success"] is True:
        print("Started")
    else:
        print(f"{service} failed to start. Error: {data["error"]}")
else:
    print("No response")
```
I'm not sure if the `is True` is necessary at all, for this I'd have to check what the success value can be at all.

Reply:
PS it's quite irritating that 3 ways of returning `None` are exhibited in the functions in this file: `return None`, just `return`, and finally not actively returning at all :P

Reply:
Yes this is not great! very unintuitive

Reply:
Purpose of my patch was to fix original bug without touching/refactoring code around it. I guess it would be great idea to separate concerns in different tasks/issues/PRs. Also not a fan of moving code inside multiple if-else blocks

Reply:
@linnik kk, I understand if you think it's safer to just add the lines.
I suppose there are already nested if-blocks, so I figured it's not too confusing of a change.